src/Assignment4/CardComponent/index.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+
+import styles from "./styles-card.css";
```

Comment:
Do you need styles here? Shouldn't it be only <import "./styles-card.css"> Or you're trying a CSS modular import?

Reply:
yes you are correct here only import "./styles-card.css is needed. I have removed the styles part